.github/workflows/daily-deploy-production.yml

```
@@ -96,6 +96,39 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    needs: validate-build-status
+    
+    steps:
+      - name: Build
+        run: yarn build --verbose —buildtype=vagovprod
+        timeout-minutes: 30
+
+      - name: Generate build details
+        run: |
+          cat > build/${{ BUILD_ENV }}/BUILD.txt << EOF
```

```suggestion
          cat > build/${{ env.BUILD_ENV }}/BUILD.txt << EOF
```

I think all uses of `BUILD_ENV` need to be prepended with `env` like this to access the environment variable

---

yeah true that, let me adjust them all. thanks for pointing this too.

---

made the changes.